src/multi_vector_simulator/utils/data_parser.py

```diff
@@ -261,13 +261,35 @@ def convert_epa_params_to_mvs(epa_dict):
     Returns
     -------
     dict_values: dict
-        mvs parameters
-
-    """
-
+        MVS json file, generated from EPA inputs, to be provided as MVS input
+
+    Notes
+    -----
+
+    - For `simulation_settings`: parameter `TIMESTEP` is parsed as unit-value pair, `OUTPUT_LP_FILE` always `False`.
+    - For `project_data`: parameter `SCENARIO_DESCRIPTION` is defined as placeholder string.
+    - `fix_cost` does not have to be provided, as default parameters will be set. But, aren´t they provided in EPA?
```

Comment:
```suggestion
    - `fix_cost` is not required, default value will be set if it is not provided.
```

We should add the questionning as comments on the code in the PR :
 But, aren´t the `fix_cost` provided in EPA?

Reply:
I agree. I added the question on the miroboard though, as this should be answered from EPA side.